cartoframes/carto_vl/carto.py

```diff
@@ -0,0 +1,7 @@
+from .basemap.basemaps import Basemaps
+from .map.map import Map
+from .layer.layer import Layer
+from .layer.local_layer import LocalLayer
+from .layer.query_layer import QueryLayer
+
+__all__ = ["Basemaps", "Map", "Layer", "LocalLayer", "QueryLayer"]
```

Comment:
To me it feels a bit redundant doing an import like this:
```python
from cartoframes.carto_vl.carto import Layer
```

What do you think about adding all of this to `__init__.py` instead so imports look like this to the user:
```python
from cartoframes import carto_vl

carto_vl.Map([ carto_vl.Layer('...')], ...)

# OR
from cartoframes.carto_vl import Layer, Map
Map([Layer(...), ])
```

Reply:
I do really like this idea, it looks cleaner and easier to understand. One of the things that came up in the weekly was if we should use `carto_vl` name or something else instead.

Reply:
what about this syntax for the examples?

```python
from cartoframes import carto_vl as vl
from cartoframes import CartoContext

context = CartoContext(
    base_url='https://cartovl.carto.com/', 
    api_key='default_public'
)

vl.Map(
    vl.Layer('populated_places'),
    context=context
).init()
```